perfdash/config.go

```diff
@@ -623,13 +623,24 @@ var (
 		},
 	}
 
+	watchListDescriptions = TestDescriptions{
```

Comment:
Why do we even need to introduce this? Why can't we use the existing performance?

Reply:
honesty i don't understand why it doesn't work.

Reply:
the tests use the `performance` type - https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-scalability/sig-scalability-experimental-periodic-jobs.yaml#L198 atm

Reply:
I started a slack thread if you want to chime in - https://kubernetes.slack.com/archives/C09QZTRH7/p1684911329191689

Reply:
It doesn't work due to the fact that perfdash has predefined file names here: https://github.com/kubernetes/perf-tests/pull/2276/files#diff-17340b391bfccaca6d6cf3409b032676038adfa8dead9fb15f65557a99bd098eR82
and `watch-list/config.yaml` has test name `watch-list` and not `load`

Reply:
This is... weird.

Can we then instead parametrized the map and reuse it? (i.e. make load/watch-list a parameter of function returning this map)?

Reply:
If we want to keep the new type then I don't see value in making the map reusable. IMO it would make the code less readable. 

What I would like to see is to make the whole mechanism more generic. Ideally if add a new job would not require defining new types.

Reply:
> I would appreciate if the whole mechanism could be more generic. Ideally if add a new job would not require defining new types.

I would like that too, but I don't want to overinvest in perfdash...